starters/graphql-serverless-contentful/README.md

```diff
@@ -162,18 +160,19 @@ Easily define your applications as AWS Lambda functions and their triggers throu
 
 `Serverless plugins` extend the Serverless Framework with new features. Plugins in this application include:
 
-- [`serverless-offline](https://github.com/dherault/serverless-offline)` - allows us to deploy our application locally to speed up development cycles.
-- [`serverless-plugin-typescript`](https://github.com/graphcool/serverless-plugin-typescript) - use Typescript with zero-config
-- [serverless-dotenv-plugin](https://github.com/infrontlabs/serverless-dotenv-plugin) - Preload function environment variables into Serverless.
--
+- [`serverless-offline`](https://github.com/dherault/serverless-offline) - allows us to deploy our application locally to speed up development cycles.
+- [`serverless-plugin-typescript`](https://github.com/graphcool/serverless-plugin-typescript) - allows the use of Typescript with zero-config.
+- [`serverless-dotenv-plugin`](https://github.com/infrontlabs/serverless-dotenv-plugin) - Preload function environment variables into Serverless.
 
 ### Contentful CMS
 
 Contentful CMS is a composable content management platform that meets the unique demands of digital content and all the teams that produce and work with it. It allows content creators to focus on assembling, editing, approving and publishing content.
 
-### Redis
+### Caching
```

Comment:
This section is about the technologies used. The previous topics covered go into the exact software used, where this change moves away from the software and instead shows the functionality.

Would it make sense to revert this change? The changes to the actual content of this chapter are great and can stay as is.